vortex-io/src/aligned.rs

```diff
@@ -0,0 +1,160 @@
+#![allow(dead_code)]
```

Comment:
if you don't have this and you build without enabling `compio` or `tokio`, you'll get an error

Reply:
Actually i can change this to a cgf_attr

Reply:
actually there are like 4 different conditions for when certain fields/methods aren't used and you end up doing a lot of `#[cfg(feature =` if you don't do this one, I think doing this one is just cleaner